dokka-integration-tests/cli/projects/it-cli/src/main/kotlin/RootPackageClass.kt

```diff
@@ -0,0 +1,10 @@
+@file:Suppress("unused")
+
+/**
+ * A class that lives inside the root package
+ *
+ * <me@mail.com>
```

Comment:
Some context sharing:

I'd add a comment that this input was added very intentionally and it's supposed to check for some corner cases, because someone might think it's just some random strings, and delete it / not adapt it when things are changing around here

```suggestion
 * <me@mail.com> - checks markdown parsing of emails and non-trivial cases like #3329, should compile
```

Something similar is done in https://github.com/Kotlin/dokka/blob/8e5c63d035ef44a269b8c43430f43f5c8eebfb63/dokka-integration-tests/gradle/projects/it-js-ir-0/src/main/kotlin/RootPackageClass.kt#L16-L26 after the test data became invalid and I had to find a subtitution (https://github.com/Kotlin/dokka/pull/2287#discussion_r846667111)

Reply:
Yeah, sounds good, I thought about something like this, but was worrying that it will be to adhoc :)